src/format.cc

```diff
@@ -70,6 +70,8 @@ template FMT_API wchar_t detail::decimal_point_impl(locale_ref);
 template FMT_API void detail::buffer<wchar_t>::append(const wchar_t*,
                                                       const wchar_t*);
 
+#ifdef FMT_EXPORT
```

Comment:
Why is this guard necessary here but not for the explicit instantiations above? This seems to be inconsistent.

Reply:
Also, doesn't a compilation unit always export regardless of library type? As such, a static library would instantiate the struct as well. The only case that does not require the explicit instantiation is the header only variant. But then, the instantiations in the compilation unit are irrelevant anyway.

Reply:
You're right. This guard is unnecessary.

Problem only in a ```__attribute__((visibility("default")))``` and ```extern template``` declaration on static library.